cmake/modules/FindOpenEXR.cmake

```diff
@@ -33,6 +33,18 @@ DOC
     "OpenEXR headers path"
 )
 
+find_path(IMATH_INCLUDE_DIR
+    Imath/half.h
```

Comment:
@meshula noted in an internal review here at Pixar that the find module is missing debug libraries (which are named with a trailing __d_). Because of this it won't be possible to debug in imath or exr on mac/linux and won't be able to link on windows with a debug build.

Nick suggested that OpenEXR 3 has got an OpenEXRConfig.cmake output file that takes care of this debug builds, but it'll be some time before we're able to take advantage of that, since vfx2021 is when openexr 3.1 is introduced.

Reply:
Okay this should now support looking for Imath debug files if doing a debug build or if a variable is set. @meshula does this look okay to you now?